apps/meteor/client/views/root/MainLayout/LayoutWithSidebarV2.tsx

```diff
@@ -50,7 +52,12 @@ const LayoutWithSidebarV2 = ({ children }: { children: ReactNode }): ReactElemen
 				className={[embeddedLayout ? 'embedded-view' : undefined, 'menu-nav'].filter(Boolean).join(' ')}
 			>
 				<MainLayoutStyleTags />
-				{!removeSidenav && <Sidebar />}
+				{!removeSidenav && (
```

Comment:
WDYT, since we're already here, we could invert this flag to `showSidenav`. "Negative" boolean names can be quite confusing. Specially in this check since `!removeSidenav` would translate to `notRemoveSidenav`. Otherwise, `!showSidenav` would be `notShowSidenav`, making it a little clearer and easier to understand. Just a sidenote though.